### ISSUE TITLE
Non-existing field mentioned in docs

### ISSUE THREAD

**Issue by zakarumych** [KhronosGroup/Vulkan-Docs#1280]
In the valid usage for `VkAccelerationStructureBuildOffsetInfoKHR` there is a line:

```
For geometries of type VK_GEOMETRY_TYPE_TRIANGLES_KHR, if the geometry doesn’t use indices,
the offset primitiveOffset from VkAccelerationStructureGeometryTrianglesDataKHR::vertexData
must be a multiple of the component size of
VkAccelerationStructureGeometryTrianglesDataKHR::vertexType
```

there is not field `vertexType` in `VkAccelerationStructureGeometryTrianglesDataKHR`
Maybe it must be `component size of vertexFormat`? Or just `vertexStride`?

**Comment by dgkoch** [comment]
I believe it should be: 

must be a multiple of the component size of VkAccelerationStructureGeometryTrianglesDataKHR::vertexFormat

**Comment by dgkoch** [comment]
This was fixed in the 1.2.145 spec update.
